src/Point/PointToInterlockingConnectionB4R1Impl.cs

```diff
@@ -0,0 +1,153 @@
+using EulynxLive.Messages.Baseline4R1;
+using EulynxLive.Point;
+using Google.Protobuf;
+using Grpc.Core;
+using Grpc.Net.Client;
+using Sci;
+using EulynxLive.Messages.IPointToInterlockingConnection;
+using static EulynxLive.Messages.IPointToInterlockingConnection.IPointToInterlockingConnection;
+using static Sci.Rasta;
+
+public class PointToInterlockingConnectionB4R1Impl : IPointToInterlockingConnection
+{
+    private readonly ILogger _logger;
+    private readonly string _localId;
+    private readonly string _localRastaId;
+    private readonly string _remoteId;
+    private readonly string _remoteEndpoint;
+    AsyncDuplexStreamingCall<SciPacket, SciPacket>? _currentConnection;
```

Comment:
I thought contrary to our previous discussion it would be great to wrap the gRPC connection into a lightweight interface:

```csharp
interface IConnection {
   Task<byte[]> ReceiveAsync();
   Task SendAsync(byte[] bytes);
}
```